diff --git a/battle.php b/battle.php
index 13c114c..4026f3e 100644
--- a/battle.php
+++ b/battle.php
@@ -63,12 +63,11 @@ $outcome = battle($ship1, $ship1Quantity, $ship2, $ship2Quantity);
                     <?php echo $ship2Quantity; ?> <?php echo $ship2->getName(); ?><?php echo $ship2Quantity > 1 ? 's': ''; ?>
                 </p>
             </div>
-            <?php var_dump($outcome); ?>
             <div class="result-box center-block">
                 <h3 class="text-center audiowide">
                     Winner:
                     <?php if ($outcome['winning_ship']): ?>
-                        <?php echo $outcome['winning_ship']['name']; ?>
+                        <?php echo $outcome['winning_ship']->getName(); ?>
                     <?php else: ?>
                         Nobody
                     <?php endif; ?>
@@ -77,11 +76,11 @@ $outcome = battle($ship1, $ship1Quantity, $ship2, $ship2Quantity);
                     <?php if ($outcome['winning_ship'] == null): ?>
                         Both ships destroyed each other in an epic battle to the end.
                     <?php else: ?>
-                        The <?php echo $outcome['winning_ship']['name']; ?>
+                        The <?php echo $outcome['winning_ship']->getName(); ?>
                         <?php if ($outcome['used_jedi_powers']): ?>
                             used its Jedi Powers for a stunning victory!
                         <?php else: ?>
-                            overpowered and destroyed the <?php echo $outcome['losing_ship']['name'] ?>s
+                            overpowered and destroyed the <?php echo $outcome['losing_ship']->getName() ?>s
                         <?php endif; ?>
                     <?php endif; ?>
                 </p>
