diff --git a/play.php b/play.php
index 728745d..ecda441 100644
--- a/play.php
+++ b/play.php
@@ -40,6 +40,12 @@ class Ship
             );
         }
     }
+
+    public function doesGivenShipHaveMoreStrength($givenShip)
+    {
+        return $givenShip->strength > $this->strength;
+    }
+
 }
 
 /**
@@ -70,3 +76,10 @@ $otherShip->strength = 50;
 
 echo '<hr/>';
 printShipSummary($otherShip);
+echo '<hr/>';
+
+if ($myShip->doesGivenShipHaveMoreStrength($otherShip)) {
+    echo $otherShip->name.' has more strength';
+} else {
+    echo $myShip->name.' has more strength';
+}
