diff --git a/functions.php b/functions.php
index 4876bb7..236ad4a 100644
--- a/functions.php
+++ b/functions.php
@@ -7,9 +7,9 @@ function get_ships()
     $ships = array();
 
     $ship = new Ship();
-    $ship->name = 'Jedi Starfighter';
-    $ship->weaponPower = 5;
-    $ship->jediFactor = 15;
+    $ship->setName('Jedi Starfighter');
+    $ship->setWeaponPower(5);
+    $ship->setJediFactor(15);
     $ship->setStrength(30);
     $ships['starfighter'] = $ship;
 
diff --git a/index.php b/index.php
index af5c34e..5069fc7 100644
--- a/index.php
+++ b/index.php
@@ -66,8 +66,8 @@ if (isset($_GET['error'])) {
                     <?php foreach ($ships as $ship): ?>
                         <tr>
                             <td><?php echo $ship->getName(); ?></td>
-                            <td><?php echo $ship->weaponPower; ?></td>
-                            <td><?php echo $ship->jediFactor; ?></td>
+                            <td><?php echo $ship->getWeaponPower(); ?></td>
+                            <td><?php echo $ship->getJediFactor(); ?></td>
                             <td><?php echo $ship->getStrength(); ?></td>
                         </tr>
                     <?php endforeach; ?>
diff --git a/lib/Ship.php b/lib/Ship.php
index 71cc61f..770306e 100644
--- a/lib/Ship.php
+++ b/lib/Ship.php
@@ -2,11 +2,11 @@
 
 class Ship
 {
-    public $name;
+    private $name;
 
-    public $weaponPower = 0;
+    private $weaponPower = 0;
 
-    public $jediFactor = 0;
+    private $jediFactor = 0;
 
     private $strength = 0;
 
@@ -60,4 +60,43 @@ class Ship
         return $givenShip->strength > $this->strength;
     }
 
+    /**
+     * @return int
+     */
+    public function getWeaponPower()
+    {
+        return $this->weaponPower;
+    }
+
+    /**
+     * @return int
+     */
+    public function getJediFactor()
+    {
+        return $this->jediFactor;
+    }
+
+    /**
+     * @param string $name
+     */
+    public function setName($name)
+    {
+        $this->name = $name;
+    }
+
+    /**
+     * @param int $weaponPower
+     */
+    public function setWeaponPower($weaponPower)
+    {
+        $this->weaponPower = $weaponPower;
+    }
+
+    /**
+     * @param int $jediFactor
+     */
+    public function setJediFactor($jediFactor)
+    {
+        $this->jediFactor = $jediFactor;
+    }
 }
