diff --git a/functions.php b/functions.php
index 236ad4a..2b4a457 100644
--- a/functions.php
+++ b/functions.php
@@ -13,6 +13,27 @@ function get_ships()
     $ship->setStrength(30);
     $ships['starfighter'] = $ship;
 
+    $ship2 = new Ship();
+    $ship2->setName('CloakShape Fighter');
+    $ship2->setWeaponPower(2);
+    $ship2->setJediFactor(2);
+    $ship2->setStrength(70);
+    $ships['cloakshape_fighter'] = $ship2;
+
+    $ship3 = new Ship();
+    $ship3->setName('Super Star Destroyer');
+    $ship3->setWeaponPower(70);
+    $ship3->setJediFactor(0);
+    $ship3->setStrength(500);
+    $ships['super_star_destroyer'] = $ship3;
+
+    $ship4 = new Ship();
+    $ship4->setName('RZ-1 A-wing interceptor');
+    $ship4->setWeaponPower(4);
+    $ship4->setJediFactor(4);
+    $ship4->setStrength(50);
+    $ships['rz1_a_wing_interceptor'] = $ship4;
+
     return $ships;
 
     /*
